flex/validation/parameter.py

```diff
@@ -125,7 +125,10 @@ def construct_parameter_validators(parameter, context):
     if 'schema' in parameter:
         schema_validators = construct_schema_validators(parameter['schema'], context=context)
         for key, value in schema_validators.items():
-            validators.setdefault(key, value)
+            if key in validators:
+                validators[key] += value
```

Comment:
I'm having trouble following the logic behind this change which implies this section could use a comment explaining what is happening and why.  Is that something you can add?